include/gatekeeper_config.h

```diff
@@ -17,6 +17,7 @@
  */
 
 #include <lua.h>
+#include <netinet/in.h>
```

Comment:
1. This include doesn't seem to be needed in this header file.

2. It seems that we need a include for `struct timeval`.